src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/PadUntypedObjectTests.cs

```diff
@@ -45,9 +45,51 @@ public void PadUntypedObjectTest()
             Assert.Equal(FormulaType.UntypedObject, fv3.Type);
             Assert.True(fv3 is BlankValue);
         }
+
+        [Fact]
```

Comment:
We should add test with ParseJson(). 

Ie, what does this do?
Set(ParseJson("{x:5}").x, 99)

Reply:
`Error 22-24: The value passed to the 'Set' function cannot be changed.`